.env.development

```diff
@@ -1,11 +1,13 @@
 # App settings
-export DISABLE_DASHBOARD_REFRESH=true
+DISABLE_DASHBOARD_REFRESH=true
 
 # Database settings
 DEFENCE_SOLICITOR_DATABASE_HOST=localhost
 DEFENCE_SOLICITOR_DATABASE_PASSWORD=""
 DEFENCE_SOLICITOR_DATABASE_USERNAME="postgres"
 
+FORCE_SSL=false
```

Comment:
https://github.com/ministryofjustice/defence-request-service/blob/master/config/environments/production.rb#L46 but not in https://github.com/ministryofjustice/defence-request-service/blob/master/config/environments/development.rb